client/plots/matrix.controls.js

```diff
@@ -429,7 +432,10 @@ export class MatrixControls {
 								inputs: [
 									{ settingsKey: 'colspace', min: 0, max: 20, step: 1 },
 									{ settingsKey: 'rowspace', min: 0, max: 20, step: 1 }
-								]
+								],
```

Comment:
@congyu-lu Users may prefer wider gap between heatmap columns, especially in small cohorts. If a heatmap should not have column spacing, then the default value should be 0.

Reply:
similar to rowh changing these will make the dendrograms misaligned. should we disable it for now before it's fixed?